docs/configuration.md

```diff
@@ -202,6 +202,15 @@ of the most common options to set are:
   </td>
   <td>2.3.0</td>
 </tr>
+<tr>
+  <td><code>spark.driver.minMemoryOverhead</code></td>
+  <td>384Mib</td>
```

Comment:
@tgravescs just want to make sure I'm being correct here.
I create this flag with "  .createWithDefaultString("384m")". Is it correct for me to write here `Mib` instead of `m` for clarity to the reader or should I stick to exactly the value it contains?